tests/decorators/test_decorators.py

```diff
@@ -211,17 +211,18 @@ def dummy():
 
         func = self._get_func(app)
 
-        trigger = func.get_bindings()[0]
+        output = func.get_bindings()[0]
 
-        self.assertEqual(trigger.get_dict_repr(), {
+        self.assertEqual(len(func.get_bindings()), 1)
```

Comment:
minor: put length check before output to not cause null pointer exception, else looks good!

Reply:
updated Thanks.